gradle/libs.versions.toml

```
@@ -10,6 +10,7 @@ androidx-espresso = "3.7.0"
 androidx-lifecycle = "2.9.4"
 androidx-testExt = "1.3.0"
 composeMultiplatform = "1.9.0"
+composeMultiplatformVersion = "2.9.0"
```

variable name is missleading, maybe rename it to kizitonwoseCalendarVersion

```suggestion
kizitonwoseCalendarVersion = "2.9.0"
```